src/angular/projects/spark-angular/package.json

```diff
@@ -20,7 +20,7 @@
     "@angular/platform-browser": "^6.1.4",
     "@angular/platform-browser-dynamic": "^6.1.4",
     "@angular/router": "^6.1.4",
-    "@sparkdesignsystem/spark": "^10.0.1",
+    "@sparkdesignsystem/spark-core": "^10.0.0",
```

Comment:
New one should be 'spark' not 'spark-core'.

Reply:
fixed